### ISSUE TITLE
crawler question

### ISSUE THREAD

**Issue by silver2345** [sergiotapia/magnetissimo#61]
does the crawler update or override the seeders & leechers count in the database when it crawls the same torrent day after day?

**Comment by sergiotapia** [comment]
No, it doesn't update the torrent in any way.

**Comment by silver2345** [comment]
thanks. what if i set the crawl to only cache torrents with 200 or more seeders. if the torrent only has 100 seeders but later reaches 2000 seeders, will the crawler cache it the second time around? or will it ignore the torrent altogether?

**Comment by silver2345** [comment]
@sergiotapia, sorry, dont know how to reopen issues

**Comment by sergiotapia** [comment]
I reopened the issue, I closed it because I thought it answered your question.

It'll crawl the torrent once and only once. If you caught it brand new with 10 leechers, even if tomorrow it has 10000 leechers, Magnetissimo won't update it.

**Comment by silver2345** [comment]
oh, thats unfortunate. would it be difficult to teach the crawler to update the counts? seeders and leechers change from day to day, it makes sense to constantly update the database. is there a particular reason you didnt design it to constantly update seed/leech counts?

**Comment by Elly6** [comment]
I agree, updating seeder/leecher counters makes sense. Torrents vary in popularity from day to day pretty dramatically. @sergiotapia, would updating the crawler to do so be a simple task for you?

**Comment by sergiotapia** [comment]
Yeah it would be trivial. I'll include that for you guys, thinking about it last night it would be a great feature to have.

**Comment by Elly6** [comment]
Wow, thanks so much. So we're looking at big update soon?

- crawlers for torrentz2/torrentproject
- ability to Sort from within the browser
- crawlers updating seed/leech count in database
- improved crawlers
- more?

**Comment by sergiotapia** [comment]
Yep! 😄

**Comment by tchoutri** [comment]
Now that we're using RSS feeds this will be much more difficult. Perhaps adding a "force reload" feature for one or more links would be a plus but I don't think it's a priority now :)

**Comment by sergiotapia** [comment]
Definitely not a priority
